packages/ts/src/rules/withStatements.test.ts

```diff
@@ -0,0 +1,28 @@
+import { ruleTester } from "./ruleTester.js";
+import rule from "./withStatements.js";
+
+ruleTester.describe(rule, {
+	invalid: [
+		{
+			code: `
+with (ooo.eee.oo.ah_ah.ting.tang.walla.walla) {
```

Comment:
😄 As much as I love 50s novelty song references, I've been trying to keep rule test variables consistently plain. Fancy things like this can be confusing to read - especially for folks who don't get the reference.

```suggestion
with (value) {
```

Reply:
> Had no idea this was a reference to something (I do love 50s music though!)

I might have to take back liking 50s music... I now have regrets in my life... 😅